etc/config-template.toml

```diff
@@ -158,6 +158,18 @@
 ## Otherwise, the OOM problem might occur in TiKV.
 # capacity = "1GB"
 
+[storage.gc]
+## The threshold of redundant ratio in a region to trigger GC.
+## The default value 1.1 should work for most cases. Set it to a value less than 1.0 to always trigger GC.
+# ratio-threshold = 1.1
+
+## The number of keys to GC in one batch.
+# batch-size = 512
```

Comment:
Consider try to imply in the name that the batch size is by key, not bytes.
For example,  batch-keys or max-keys-in-batch or something